### ISSUE TITLE
Battle Gear 4 Tuned : Mislabeled input

### ISSUE THREAD

**Issue by nixxou** [teknogods/TeknoParrotUI#960]
![image](https://github.com/user-attachments/assets/1d42111f-9341-4962-9144-f467f15256fe)

It should be "Gear 4 / Shift Up" instead of "Gear 3 / Shift Up" and "Gear 3 / Shift Down" instead of "Gear 4 / Shift Down"

**Comment by Riddler1** [comment]
This is correct (as i found out). The "shift up/down" actually refers to the menu selector and not gear shift
